packages/swapper/src/swappers/JupiterSwapper/utils/constants.ts

```diff
@@ -0,0 +1,11 @@
+import type { ChainId } from '@shapeshiftoss/caip'
+import { KnownChainIds } from '@shapeshiftoss/types'
+
+import type { SupportedChainIds } from '../../../types'
+
+export const jupiterSupportedChainIds = [KnownChainIds.SolanaMainnet] as ChainId[]
```

Comment:
nitpick: 
```suggestion
export const jupiterSupportedChainIds: [KnownChainIds.SolanaMainnet] = [KnownChainIds.SolanaMainnet]
```

Reply:
Yes or we will have some issues with types deeper in the tree (try it out if you want)